champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/bow/NapalmArrow.java

```diff
@@ -0,0 +1,267 @@
+package me.mykindos.betterpvp.champions.champions.skills.skills.ranger.bow;
+
+import com.destroystokyo.paper.ParticleBuilder;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import me.mykindos.betterpvp.champions.Champions;
+import me.mykindos.betterpvp.champions.champions.ChampionsManager;
+import me.mykindos.betterpvp.champions.champions.skills.data.SkillActions;
+import me.mykindos.betterpvp.champions.champions.skills.types.FireSkill;
+import me.mykindos.betterpvp.champions.champions.skills.types.OffensiveSkill;
+import me.mykindos.betterpvp.champions.champions.skills.types.PrepareArrowSkill;
+import me.mykindos.betterpvp.core.combat.events.CustomDamageEvent;
+import me.mykindos.betterpvp.core.combat.throwables.ThrowableItem;
+import me.mykindos.betterpvp.core.combat.throwables.ThrowableListener;
+import me.mykindos.betterpvp.core.components.champions.Role;
+import me.mykindos.betterpvp.core.components.champions.SkillType;
+import me.mykindos.betterpvp.core.framework.updater.UpdateEvent;
+import me.mykindos.betterpvp.core.listener.BPvPListener;
+import me.mykindos.betterpvp.core.utilities.UtilDamage;
+import me.mykindos.betterpvp.core.utilities.UtilServer;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.Particle;
+import org.bukkit.Sound;
+import org.bukkit.World;
+import org.bukkit.entity.ArmorStand;
+import org.bukkit.entity.Arrow;
+import org.bukkit.entity.Item;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.block.Action;
+import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.event.entity.EntityShootBowEvent;
+import org.bukkit.event.entity.PlayerDeathEvent;
+import org.bukkit.event.entity.ProjectileHitEvent;
+import org.bukkit.event.player.PlayerQuitEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.scheduler.BukkitRunnable;
+import org.bukkit.util.Vector;
+
+import java.util.*;
+
+@Singleton
+@BPvPListener
+public class NapalmArrow extends PrepareArrowSkill implements ThrowableListener, FireSkill, OffensiveSkill {
+
+    private double baseBurnDuration;
+    private double burnDurationIncreasePerLevel;
+    private double baseDamage;
+    private double damageIncreasePerLevel;
+    private double baseDuration;
+    private double durationIncreasePerLevel;
+    private double velocityMultiplier;
+    private double yComponentVelocityMultiplier;
+    private int damageDelay;
+    private int numFlames;
+    private final Map<UUID, Arrow> napalmArrows = new HashMap<>();
+    private final Random random = new Random();
```

Comment:
Use UtilMath.RANDOM, no need to create a new one tbh

Reply:
sonar cloud yelled at me to do this